platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/LogoutState.java

```diff
@@ -97,4 +103,22 @@ public String getCurrentRequestId() {
   public void setCurrentRequestId(String currentRequestId) {
     this.currentRequestId = currentRequestId;
   }
+
+  public List<String> getSessionIndexes() {
+    return sessionIndexes;
+  }
+
+  @SuppressWarnings("unused")
```

Comment:
❓ If we need this `@SuppressWarnings`, we may want to add a comment explaining why.

Reply:
It is used, but it is called by blueprint and IntelliJ doesn't account for that.